readme.md

```diff
@@ -158,7 +159,7 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
 
-* **Your Answer:** 
+* **Your Answer:** The _id field is the one that will throw an error as this is included by Mondo by default. We can get around thos by setting _id = 0 in the projection.
```

Comment:
You should encounter an error when you run `.create().select()`. You will have to re-request the information or use the `.save()` method to return the updated record.